src/aggregators/rssa.jl

```diff
@@ -119,10 +117,8 @@ end
     if nomorejumps!(p, sum_rate)
         return nothing
     end
-
     # next jump type
     @unpack ma_jumps, rates, cur_rate_high, cur_rate_low, rng = p
-    #rerl        = one(sum_rate)
```

Comment:
Please don't remove the commented out lines in the RSSA file here and below. This is something we want to be able to retest periodically, and it is easier to just leave the three commented lines...

Reply:
Got it